CWE/events/burma.txt

```diff
@@ -174,6 +174,22 @@ country_event = {
 				sphere_owner = { diplomatic_influence = { who = RAK value = 200 } }
 			}
 		}
+		random_owned = {
+			limit = { owner = { is_possible_vassal = MOS } }
+			owner = {
+				release_vassal = MOS
```

Comment:
I believe MOS is a typo

Reply:
It's not, that's the actual tag for Mon State
`1343=
{
	name="Moulmein"
	owner="BUR"
	controller="BUR"
	core="BUR"
	core="MOS"
	garrison=100.000`